src/test/ui/associated-types/trait-with-supertraits-needing-sized-self.stderr

```diff
@@ -7,8 +7,7 @@ LL | trait ArithmeticOps: Add<Output=Self> + Sub<Output=Self> + Mul<Output=Self>
 note: required by a bound in `Add`
   --> $SRC_DIR/core/src/ops/arith.rs:LL:COL
    |
-LL | pub trait Add<Rhs = Self> {
-   |               ^^^^^^^^^^ required by this bound in `Add`
+   = note: required by this bound in `Add`
```

Comment:
Should we special case the wording and make it be "required by **a** bound in `Add`"?

Reply:
oh yea definitely. there are improvements needed here, but all of these are preexisting in the sense of "this is how they look to users already"

Reply:
We already check of `is_dummy()`, we might want to modify that method to behave more as if it were `is_printable()` or something, because it is used quite a bit.